client/verta/Makefile

```diff
@@ -1,11 +1,19 @@
 # build wheel and source distributions
 build: clean
-	python setup.py sdist bdist_wheel --universal
+	python3 setup.py sdist bdist_wheel --universal
 
 # clean previously-created builds
 clean:
 	rm -rf build dist verta.egg-info
 
 # upload distributions to PyPI
-upload: build
-	python -m twine upload dist/* -u \${PYPI_USERNAME} -p \${PYPI_PASSWORD}
+upload:
+	# checking TWINE_USERNAME and TWINE_PASSWORD env vars
+	python3 -m twine upload --non-interactive dist/*
+
+# tag the current commit using the client's version number
+tag: VERSION=$$(python3 -c 'import verta; print(verta.__version__)')
+tag: TAG="client-v$(VERSION)"
+tag:
```

Comment:
I'm confused by having the target defined 3 times

Reply:
Oh no I was worried that I was accidentally using something obscure haha

The first two are [target-specific variable](https://www.gnu.org/software/make/manual/make.html#Target_002dspecific) assignments. It doesn't re-define `tag`—rather, it defines variables `VERSION` and `TAG` solely for use in the `tag` target.